src/meshapi/templates/widgets/external_link.html

```diff
@@ -0,0 +1,16 @@
+{% load i18n static %}
+<div class="related-widget-wrapper">
+    {% include "django/forms/widgets/input.html" %}
+    {% block links %}
+    {% spaceless %}
+    <a class="related-widget-wrapper-link view-related"
+        {% if widget.value %}
+            href="{{ widget.formatted }}"
+        {% endif %}
```

Comment:
Ux nit: make this if block control a CSS style instead, so the icon is like 50% opacity instead of missing

Reply:
Not sure I understand. This is what it looks like when the value is missing: 
![image](https://github.com/user-attachments/assets/3a315f87-5729-4e13-afec-d7c01964028e)

Reply:
Oh nvm looks great